### DISCUSSION TITLE
About Twemoji Uses

### DISCUSSION THREAD

**Discussion by ywbird** [mmistakes/minimal-mistakes#3171] [Q&A] [answered]
I tried to use twemoji in minimal mistakes use jekyll-twemoji plugin.
On localhost it WORKED!
But on my github page it is NOT WORKING...
What can I do?
(Currently Im add Jemoji plugin..)

**Comment by mmistakes** [comment, marked as answer]
3rd party gems are not supported on GitHub Pages, this is why it worked locally for you.
You'll need to use a GitHub action to build your Jekyll site instead of using it's built in build process. Then you can install any 3rd party plugins or gems you wish.

ref: https://jekyllrb.com/docs/continuous-integration/github-actions/
